common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -691,6 +679,39 @@ class CelebornConf(loadDefaults: Boolean) extends Cloneable with Logging with Se
   def rpcCacheConcurrencyLevel: Int = get(RPC_CACHE_CONCURRENCY_LEVEL)
   def rpcCacheExpireTime: Long = get(RPC_CACHE_EXPIRE_TIME)
   def pushDataRpcTimeoutMs = get(PUSH_DATA_RPC_TIMEOUT)
+  def registerShuffleRpcAskTimeout: RpcTimeout = {
+    get(REGISTER_SHUFFLE_RPC_ASK_TIMEOUT).map { timeout =>
+      new RpcTimeout(
+        timeout.milli,
+        REGISTER_SHUFFLE_RPC_ASK_TIMEOUT.key)
+    }.getOrElse {
+      new RpcTimeout(
+        rpcAskTimeout.duration * (reserveSlotsMaxRetries + 2),
```

Comment:
> why use such default value?

Since  reserve slot can cost 1 rpcAskTimeout and it can retry `reserveSlotsMaxRetries` times, before retry reserve slot, LifecycleManager need ask master for `releaseSlot`.